lib/components/user/places/favorite-place-screen.js

```diff
@@ -101,6 +101,10 @@ class FavoritePlaceScreen extends Component {
     // Get the places as shown (and not as retrieved from db), so that the index passed from URL applies
     // (indexes 0 and 1 are for Home and Work locations).
     const place = this._getPlaceToEdit(loggedInUser)
+    // For <input> controls, the value must be at least ''.
+    if (place && place.name === null) {
```

Comment:
We can probably do 
```suggestion
    if (place?.name === null) {
```

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/337d3b0a77ffbf0322683910f3e1647f79dc4d6b